packages/atomic/src/global/global.pcss

```diff
@@ -23,47 +23,47 @@
 
 @layer components {
   .link {
-    @apply text-primary hover:text-primary-light hover:underline focus:text-primary-light focus:underline;
+    @apply text-primary hover:text-primary-light hover:underline focus-visible:text-primary-light focus-visible:underline;
```

Comment:
I'm wondering if there's something we could do to either warn or auto correct any `.pcss` file that would use `focus:` instead of `focus:visible` down the line ?

Reply:
Investigated removing the variant, doesn't seem possible with tailwind, only possible with utilities.
Also investigated creating a Tailwind plugin that overrides focus to focus-visible, it didn't allow me, crashing the build